### DISCUSSION TITLE
Version 8.2.0.19 Ignoring Setting to Use Browser to Download in Chrome

### DISCUSSION THREAD

**Discussion by jkpolk123** [aclap-dev/video-downloadhelper#73] [Help and Q&A]
### Your question

With the latest update this morning, I am forced to only use the companion app to download which is not what I want.  I have tried reloading the extension, updated chrome to the latest version, changed the settings under download processor to Ask, Companion APp, and Browser but in all cases downloads are going straight to the companion app.   I like the ability to change the name to something else and the companion app does not provide this option.

version: 8.2.0.19
channel: stable
build date: 2024-02-11

**Comment by paulrouget** [comment]
> I like the ability to change the name to something else

Do you use Quick Download, but regular "Download":

<img width="265" alt="Screenshot 2024-02-12 at 23 04 13" src="https://github.com/aclap-dev/video-downloadhelper/assets/373579/c3a2dccc-1356-4f53-8038-37985542269f">

**Comment by jkpolk123** [comment]
Thanks.  I right click on the icon, then click on VideoDownloadHelper which shows me the hits.  I click on the one that I want and this used to then bring up the dialog box to input my own name and download via the browser.   When I click the name of the video now it immediately starts in the companion app.   

I found where the option to download to hard drive you mention is.  That does bring up the box to use my own name but it's still using the companion app to download.  Thank you for at least helping solve a portion of my issue.

**Comment by dtimekeeper** [comment]
Same issue, I would prefer to use the browser to download because it's been working fine, and integrates with the browser's download manager, etc, and my network shares.

**Comment by paulrouget** [comment]
Understood!

I will be working on a fix asap. Hope to get that fixed soon.

**Reply by dtimekeeper** [reply]
thank you! you are the best!

**Comment by paulrouget** [comment]
See https://github.com/aclap-dev/video-downloadhelper/discussions/82

**Reply by dtimekeeper** [reply]
thank you for the quick fix and deploy. The Download button under Quick Download no longer seems to pop up a file save dialog where I can rename the file like last time - probably because of the previous default action, but it does save it using the browser method perfectly.

**Reply by GrampaWildWilly** [reply]
It is possible that you have to set your default action again.  I had an update recently that forgot my previously set default action.  I just set it again & it's held since then.

**Reply by dtimekeeper** [reply]
Thanks, I did a full preferences reset, but it's still doing the same thing for the Download button.
